readme.md

```diff
@@ -22,7 +22,7 @@ var args = [
 	'--strip-iptc',
 	'--strip-icc',
 	'--all-progressive',
-	'--dest=build'
+	'--dest=build',
```

Comment:
Could you create a separate commit for this change? `Add a missing comma to the example script`